### ISSUE TITLE
comment delete model

### ISSUE THREAD

**Issue by tejxv** [tejxv/my-first-blog#2]
defining a model to delete a comment which is either by a request user or anonymous user

**Comment by pratik-londhe4** [comment]
Admin can  delete comments from admin panel even from anonymous users ! , I am very new to python.

**Comment by tejxv** [comment]
yeah that's an intended behaviour, admins are superusers and they can delete comments, even from anonymous users.
and that's very good! :)
